controls/os_spec.rb

```diff
@@ -228,3 +228,19 @@
     it { should be_empty }
   end
 end
+
+control 'os-10' do
+  impact 1.0
+  title "CIS: Disable unused filesystems"
+  desc "1.1.1 Ensure mounting of cramfs, freevxfs, jffs2, hfs, hfsplus, squashfs, udf, FAT"
+  describe file('/etc/modprobe.d/CIS.conf') do
```

Comment:
@chris-rock  The above test ensures that unused filesystems  are disabled. This test ensures that mount of unused filesystem such as cramfs etc. You may want to have a look at CIS_Distribution_Independent_Linux_Benchmark_v1.0.1 Section 1.1.1 Disable unused filesystems.